packages/web-components/src/components/va-button-pair/va-button-pair.tsx

```diff
@@ -110,6 +118,31 @@ export class VaButtonPair {
     this.secondaryClick.emit(e);
   };
 
+  componentDidLoad() {
+    this.reorderButtons();
+  }
+
+  reorderButtons() {
+    let container;
+    let buttons;
+
+    const groupSelector = this.uswds ? '.usa-button-group' : '.button-group';
+    const buttonSelector = this.uswds ? '.usa-button-group__item' : 'va-button';
+
+    container = this.el.shadowRoot.querySelector(groupSelector);
+    buttons = Array.from(container.querySelectorAll(buttonSelector));
+
+    const reverseAndAppendButtons = () => {
+      buttons.reverse();
+      buttons.forEach(button => container.appendChild(button));
+    };
+
+    if ((window.innerWidth <= 480 && !this.isReordered) || (window.innerWidth > 480 && this.isReordered)) {
```

Comment:
This can't be done with media queries in CSS? For example:

```css
@media screen and (max-width: 480px) {
 :host(:not([uswds])) .button-group {
    flex-direction: column-reverse;
  }
}
```